hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1554,6 +1614,15 @@ func defaultAndValidateConfigManifest(manifest []byte) ([]byte, error) {
 		manifest = buff.Bytes()
 	case *v1alpha1.ImageContentSourcePolicy:
 	case *mcfgv1.KubeletConfig:
+		if obj.Labels == nil {
```

Comment:
nit: extract this code into a function that takes a runtime.Object and invoke it for both MachineConfig and KubeletConfig.
And actually it might make sense for ContainerRuntimeConfig as well.

Reply:
I thought about it when I saw I had to repeat code ... 
The thing is for the labels part I need a `metav1.ObjectMeta` and for the encoding a `runtime.Object`, so I would end creating two different functions and I did not know if that would improve code readability or not ...

Reply:
I did it with two different functions in the end. 
Let me know what do you think